modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -99,10 +99,68 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} <x.y[.z]> also includes the following improvement<s>:
 
-=== <TINY-vwxyz 1 changelog entry>
-//#TINY-vwxyz1
-
-// CCFR here.
+[[a-new-trigger-property-for-block-text-pattern-configurations-allowing-pattern-activation-with-either-space-or-enter-keys]]
+=== A new `trigger` property for block text pattern configurations, allowing pattern activation with either Space or Enter keys
+//#TINY-10324
+
+This release introduces a new `trigger` property for block text pattern configurations, allowing pattern activation with either `Space` or `Enter` keys.
+
+In {productname} 7.0 the default preconfigured text patterns **was changed from**:
+
+[source, js]
+----
+[     
+  { start: '*', end: '*', format: 'italic' },
+  { start: '**', end: '**', format: 'bold' },
+  { start: '#', format: 'h1', trigger: 'enter' },
+  { start: '##', format: 'h2', trigger: 'enter' },
+  { start: '###', format: 'h3', trigger: 'enter' },
+  { start: '####', format: 'h4', trigger: 'enter' },
+  { start: '#####', format: 'h5', trigger: 'enter' },
+  { start: '######', format: 'h6', trigger: 'enter' },
+  { start: '1. ', cmd: 'InsertOrderedList', trigger: 'enter' },
+  { start: '* ', cmd: 'InsertUnorderedList', trigger: 'enter' },
+  { start: '- ', cmd: 'InsertUnorderedList', trigger: 'enter' },
+]
```

Comment:
This is the example of how to achieve the previous behaviour for block text patterns, but not how they were configured. Previous default configuration did not contain the `trigger` property.
```js
[
  { start: '*', end: '*', format: 'italic' },
  { start: '**', end: '**', format: 'bold' },
  { start: '#', format: 'h1' },
  { start: '##', format: 'h2' },
  { start: '###', format: 'h3' },
  { start: '####', format: 'h4' },
  { start: '#####', format: 'h5' },
  { start: '######', format: 'h6' },
  // The following text patterns require the `lists` plugin
  { start: '1. ', cmd: 'InsertOrderedList' },
  { start: '* ', cmd: 'InsertUnorderedList' },
  { start: '- ', cmd: 'InsertUnorderedList' }
]
```

Reply:
The whole [Block patterns](http://docs-feature-70-doc-2270tiny-10324.staging.tiny.cloud/docs/tinymce/latest/content-behavior-options/#block-patterns) section should be rewritten according the new changes. The Note there is not a true anymore. 
> Block patterns are only executed on Enter, not on pressing the spacebar.
Now this is configurable.

Also, we need to rewrite the [examaple](http://docs-feature-70-doc-2270tiny-10324.staging.tiny.cloud/docs/tinymce/latest/content-behavior-options/#example-using-block-patterns) to include the new `trigger` property.

Reply:
@vpyshnenko Please re-review the new changes based on the above and let me know if this covers what your suggestions please.

Reply:
It is almost there. I've made a couple of commits to this PR by myself since it is difficult to explain all the nuances.

Reply:
Thanks @vpyshnenko appreciate the commits.